contributors/devel/conformance-tests.md

```diff
@@ -34,6 +34,7 @@ especially in the Nucleus or Application layers as described
 (example: the default list of admission plugins should not have to be tweaked for passing conformance).
 - cannot rely on any binaries that are not required for the
 linux kernel or for a kubelet to run (i.e. git)
+- container images used in the test should support multiple architectures (should not be limited to just amd64)
```

Comment:
I am 100% fine with the spirit of this.  If Brian wants to tighten up the phrasing, maybe 
"any container images used in the test must support all architectures for which kubernetes releases are built"

Reply:
I like Aaron's phrasing. If it's changed to that, then LGTM

Reply:
On it!